src/sign-verify/clu_dgst_setup.c

```diff
@@ -314,6 +394,55 @@ int wolfCLU_dgst_setup(int argc, char** argv)
         }
     }
 
+    /* create the signature if requested */
+    if (ret == WOLFCLU_SUCCESS && signing == 1) {
+        WC_RNG rng;
+
+        wc_InitRng(&rng);
```

Comment:
Please add checking or return code on call to `wc_InitRng`. Lots of work in that call.